ci/deps/azure-36-locale.yaml

```diff
@@ -15,7 +15,6 @@ dependencies:
 
   # pandas dependencies
   - beautifulsoup4
-  - gcsfs
```

Comment:
can you re-add fsscspec here as well as leaving these (pin the versions if needed)

Reply:
Sorry, what do you mean? It seems that the current version of boto/moto or something else is causing faulty locale-specific timestamp handling, that's why these were removed (because they are tested in the other runs)